index.html

```diff
@@ -926,10 +914,50 @@ <h2>The <code>application/vc+jwt</code> Media Type</h2>
   </table>
 </section>
 
-<section id="conformance"></section>
+<section id="vc-ld-jwt-media-type">
+  <h2><code>application/vc+ld+jwt</code></h2>
+  <p>
+    This specification registers the <code>application/vc+ld+jwt</code> Media Type specifically for
+    identifying a <a data-cite="rfc7519#section-3">JWT</a> conforming to the Verifiable Credentials JWT format in the `typ` header.
+  </p>
+  <table>
+    <tr>
+      <td>Type name: </td>
+      <td>application</td>
+    </tr>
+    <tr>
+      <td>Subtype name: </td>
+      <td>application/vc+ld+jwt</td>
```

Comment:
This subtype repeats `application` and probably shouldn't. Was this meant, instead?

```suggestion
      <td>vc+ld+jwt</td>
```

Reply:
Probably not, @TallTed  has been attaching `application/` to a lot of the references to the media types, it's possible this was merged in a group of those.